boards/arm64/qemu_cortex_a53/qemu_cortex_a53_smp_defconfig

```diff
@@ -1,7 +1,13 @@
 CONFIG_SOC_QEMU_CORTEX_A53=y
 CONFIG_BOARD_QEMU_CORTEX_A53=y
 CONFIG_ARM_ARCH_TIMER=y
-CONFIG_QEMU_ICOUNT=n
+CONFIG_ARMV8_A_NS=y
+CONFIG_QEMU_ICOUNT=y
+CONFIG_QEMU_ICOUNT_SHIFT=4
+CONFIG_QEMU_ICOUNT_SLEEP=y
```

Comment:
If we really really really do not need this, I'd leave this out.

Reply:
for the XIP is probably simply forgotten. The XIP should closely follow what's done for `qemu_cortex_a53`.